changelog.d/20240624_105125_kyle_test_mongo_host.md

```diff
@@ -0,0 +1 @@
+[Improvement] Set `EDXAPP_TEST_MONGO_HOST` env var in the openedx-dev image so that it no longer needs to be set by hand when running edx-platform unit tests (by @kdmccormick).
```

Comment:
do we need a changelog entry for this, considering this is only meant for dev/testing and wont impact the operators as such?

Reply:
Since Tutor is both an operator tool and a developer tool, I think it is good to make not of the improvements we make for developers too.